src/main/java/org/jenkinsci/plugins/gogs/GogsWebHook.java

```diff
@@ -106,8 +106,8 @@ public void doIndex(StaplerRequest req, StaplerResponse rsp) throws IOException
 
         // Get X-Gogs-Event
         String event = req.getHeader("X-Gogs-Event");
-        if (!"push".equals(event)) {
-            result.setStatus(403, "Only push event can be accepted.");
+        if (!"push".equals(event) && !"release".equals(event)) {
+            result.setStatus(403, "Only push or release events are accepted.");
```

Comment:
Can the `create` event be added as well ?
e.g.
```
if (!"push".equals(event) && !"release".equals(event) && !"create".equals(event)) {
    result.setStatus(403, "Only push, release or create events are accepted.");
```

Reply:
Can the create event be added as well ?
e.g.
```
if (!"push".equals(event) && !"release".equals(event) && !"create".equals(event) && !"pull_request".equals(event)) {
    result.setStatus(403, "Only push, release, create or pull_request events are accepted.");
```

Reply:
Same here as the above comment. Adding other events breaks things at the moment. Need to investigate how to fix this.

Reply:
@sanderv32 very interested in this work being completed and merged.  Is there something I can contribute to help?

Reply:
The whole thing broke down with this changes. Haven't looked for a long time at this change, but if you want to test please do so.